indico/modules/events/timetable/templates/display/_common.html

```diff
@@ -0,0 +1,62 @@
+{% from 'events/notes/_note.html' import render_note %}
+{% from 'attachments/_display.html' import render_attachments_folders %}
+{% from 'events/display/_common.html' import render_users %}
+
+{% macro render_description(item) %}
+    <span itemprop="description" class="contrib-description html">
+        {{ item.description | safe }}
+    </span>
+{% endmacro %}
+
+{% macro render_references(item) -%}
+    <ul class="reference-list">
+        {%- for reference in item.references: -%}
+            <li title="{% trans title=reference.reference_type.name %}External reference ({{ title }}){% endtrans %}"
+                class="icon-link">
+            {% if reference.url -%}
+                <a href="{{ reference.url }}" class="title" target="_blank">
+                    {{ reference.urn  or reference.value }}
+                </a>
+            {%- else -%}
+                <span class="title">{{ reference.urn or reference.value }}</span>
+            {%- endif %}
+            </li>
+        {%- endfor -%}
+    </ul>
+{%- endmacro %}
+
+{% macro render_time(item, timezone=none, hide_end_time=false) -%}
+    <span class="start-time">
+        {{ item.timetable_entry.start_dt | format_time(timezone=timezone) }}
+    </span>
+    {% if not hide_end_time %}
+        &rarr;
+        <span class="end-time">
+            {{ (item.timetable_entry.start_dt + item.duration) | format_time(timezone=timezone) }}
```

Comment:
`item.timetable_entry.end_dt` (shouldn't cause any extra queries since the object reference on the timetable entry should alraady be available)